Source/SwiftLintFramework/Rules/ValidDocsRule.swift

```diff
@@ -51,7 +51,9 @@ func declarationReturns(declaration: String, kind: SwiftDeclarationKind? = nil)
     guard let outsideBracesMatch = matchOutsideBraces(declaration) else {
         return false
     }
-
+    if declaration.containsString("init(") {
```

Comment:
I think the beginning space solution should be good :) 

I'm also not familiar enough with SourceKitten so im not sure if includes the ACL etc but it should be considered in my opinion.  

About guard/if, As far as I know, [the guidelines SwiftLint follows](https://github.com/github/swift-style-guide/blob/master/README.md#return-and-break-early), do suggest using guard to return early.